src/transaction/log_replication.cpp

```diff
@@ -164,13 +169,25 @@ namespace cublog
 	    break;
 	  case LOG_DBEXTERN_REDO_DATA:
 	  {
-	    log_rec_dbout_redo dbout_redo = m_reader.reinterpret_copy_and_add_align<log_rec_dbout_redo> ();
+	    const log_rec_dbout_redo dbout_redo = m_reader.reinterpret_copy_and_add_align<log_rec_dbout_redo> ();
 	    log_rcv rcv;
 	    rcv.length = dbout_redo.length;
 	    log_rv_redo_record (&thread_entry, m_reader, RV_fun[dbout_redo.rcvindex].redofun, &rcv, &m_redo_lsa, 0,
 				nullptr, m_redo_unzip);
+	    break;
 	  }
-	  break;
+	  case LOG_COMMIT:
+	    calculate_replication_delay_or_dispatch_async<struct log_rec_donetime> (

```

Comment:
Using struct before type name was required by old C, no longer required for C++.
```suggestion
	    calculate_replication_delay_or_dispatch_async<log_rec_donetime> (
```

Reply:
Done